### PR TITLE
SD-100: read asset info from the predefined Blockchain contract

### PR DESCRIPTION
**Pull request by domagojlatecki**
Implements contract API for two example contracts stored on Ropsten testnet.
Note: will be merged to `master` after `SD-101`

### CODE REVIEW

File: src/main/kotlin/com/ampnet/auditornode/persistence/model/AssetCategoryId.kt [hunk 1]
```diff
@@ -0,0 +1,5 @@
+package com.ampnet.auditornode.persistence.model
+
+import java.math.BigInteger
+
+inline class AssetCategoryId(val value: BigInteger) // TODO use `value class` instead on later Kotlin version
```
Comment (githinho) [inline src/main/kotlin/com/ampnet/auditornode/persistence/model/AssetCategoryId.kt:5]: You can use `Int` if you don't any special function from `BigIntger`.

File: src/main/kotlin/com/ampnet/auditornode/service/impl/GethAssetContractService.kt [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+import javax.inject.Inject
+import javax.inject.Singleton
+
+private val logger = KotlinLogging.logger {}
+
+@Singleton
+class GethAssetContractService @Inject constructor(
+    auditorProperties: AuditorProperties,
+    private val rpcProperties: RpcProperties,
+    rpc: EthereumRPC
```
Comment (githinho) [inline src/main/kotlin/com/ampnet/auditornode/service/impl/GethAssetContractService.kt:28]: Set `private val` first or last to improve readability.

**Review by githinho — APPROVED** [review]

File: src/main/kotlin/com/ampnet/auditornode/persistence/model/AssetCategoryId.kt (hunk 1 above)
Comment (domagojlatecki) [reply src/main/kotlin/com/ampnet/auditornode/persistence/model/AssetCategoryId.kt:5]: Generated contract API returns `BigInteger`. Using `Int` here would mean that it is possible to get an overflow.

Comment (githinho) [reply src/main/kotlin/com/ampnet/auditornode/persistence/model/AssetCategoryId.kt:5]: I get it, my bad.

**Review by josipkoncurat — APPROVED** [review]

**Pull request merged**